### PR TITLE
P:P #3 featuring Ollie Charles 0:30-0:45

### PR DESCRIPTION
**Pull request by jbcden**

### CODE REVIEW

File: 3/ollie-charles-captions.srt [hunk 1]
```diff
@@ -2050,7 +2051,7 @@ OC: So we'll start by taking the neighbors of A.
 
 474
 00:30:44,850 --> 00:30:49,770
-(inaudible) dictionary (inaudible).
+and the same dictionary [inaudible].
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2054]: Good call.

File: 3/ollie-charles-captions.srt [hunk 2]
```diff
@@ -2179,23 +2181,23 @@ in the standard library that does
 
 503
 00:32:31,820 --> 00:32:33,770
-these two things together... Concat map
+these two things together... concatMap
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2184]: :+1:

File: 3/ollie-charles-captions.srt [hunk 3]
```diff
@@ -2204,7 +2206,7 @@ I should be able to know
 
 509
 00:32:50,413 --> 00:32:52,600
-(inaudible) problems.
+[inaudible] of one of the example problems.
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2209]: This sounds to me like:

```
If I load the dictionary,
I should be able to now
try one of the example problems.
```

File: 3/ollie-charles-captions.srt [hunk 4]
```diff
@@ -2299,12 +2301,11 @@ So let's filter the dictionary
...
 531
 00:34:20,680 --> 00:34:23,000
-for stuff that's not to equal to N.
+for stuff that's not equal to N.
 
 532
 00:34:28,280 --> 00:34:31,880
-It's still having (inaudible)
-
+It's still compiling...
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2308]: Add a blank line between these two captions to keep .srt file valid.

File: 3/ollie-charles-captions.srt [hunk 5]
```diff
@@ -2326,8 +2327,8 @@ we know that it will eventually
 
 537
 00:34:52,514 --> 00:34:54,542
-deliver and so give us the right answer.
-
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt]: Add a blank line between these two captions to keep .srt file valid.

File: 3/ollie-charles-captions.srt [hunk 6]
```diff
@@ -2326,8 +2327,8 @@ we know that it will eventually
...
+deliver an answer even if it
+is the wrong answer.
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2331]: Add a blank line between these two captions to keep .srt file valid.

File: 3/ollie-charles-captions.srt [hunk 7]
```diff
@@ -2342,12 +2343,11 @@ cot to cat, so maybe if we just try cot to cat
...
 541
 00:35:03,660 --> 00:35:05,373
-see if we get that solution.
+and see if we get that solution.
 
 542
 00:35:05,533 --> 00:35:07,240
-OC: For the smallest (inaudible)
-
+OC: For the smallest thing I can think of
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2350]: Add a blank line between these two captions to keep .srt file valid.

File: 3/ollie-charles-captions.srt [hunk 8]
```diff
@@ -2406,8 +2406,7 @@ You know that you've got that one.
 
 556
 00:36:11,310 --> 00:36:15,680
-OC: (inaudible)
-
+OC: Great job!
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2409]: Add a blank line between these two captions to keep .srt file valid.

File: 3/ollie-charles-captions.srt [hunk 9]
```diff
@@ -2733,7 +2734,7 @@ ones that are the shortest.
 
 633
 00:41:44,480 --> 00:41:47,573
-OC: Right so reading the
+OC: Right so a greedier
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2737]: Good catch!

File: 3/ollie-charles-captions.srt [hunk 10]
```diff
@@ -2788,7 +2789,7 @@ and it's certainly not a good solution.
 
 646
 00:42:37,520 --> 00:42:40,770
-In the original spec of the (inaudible) it said
+In the original spec of the [inaudible] it said
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2792]: I hear:

```
In the original spec of the kata it said...
```

File: 3/ollie-charles-captions.srt [hunk 11]
```diff
@@ -2937,12 +2938,13 @@ no words at all in the chain.
 
 680
 00:44:32,560 --> 00:44:35,666
-That's not really the
-environment I want to have in the code.
+That's not really an
+invariant I want to have in the code.
```
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2942]: Good catch!

File: 4/john-wiegley-captions.srt [hunk 12]
```diff
@@ -5424,7 +5426,7 @@ work in this little scratch file)
 
 1271
 01:46:19,176 --> 01:46:20,235
-Feed should be fine.
+Phi should be fine.
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:5429]: Good catch!

File: 4/john-wiegley-captions.srt [hunk 13]
```diff
@@ -5505,11 +5507,11 @@ and we've got division.
 
 1290
 01:47:52,700 --> 01:47:53,410
-we've got vnc ....
+we've got phi and psi ....
 
 1291
 01:47:54,580 --> 01:47:57,520
-A global inaudible and a local inaudible.
+A global phi and a local psi.
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:5514]: :wink:

File: 4/john-wiegley-captions.srt [hunk 14]
```diff
@@ -5940,12 +5947,12 @@ If I go into my parser and look at
 
 1392
 01:56:36,470 --> 01:56:38,700
-this rebel function
+this repl function
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:5950]: Good catch!
:+1:

File: 4/john-wiegley-captions.srt [hunk 15]
```diff
@@ -6169,20 +6179,21 @@ evaluators get easier to write.
...
 02:00:01,760 --> 02:00:03,880
-In bc I go back
-out to do the expression
+In bc it will echo back
+out to you the expression
 
 1447
 02:00:03,880 --> 02:00:06,000
-and inaudible
+and [inaudible]
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:6196]: I hear:

```
in pretty-printed form, so I'd like to do that as well.
```

File: 4/john-wiegley-captions.srt [hunk 16]
```diff
@@ -6205,15 +6216,16 @@ where I would go with this.
...
 1452
 02:00:20,350 --> 02:00:21,880
-But if I was trying to compete
+But I don't know if I'm trying
+to compete
 
 1453
 02:00:21,880 --> 02:00:23,520
-with a mathematics professor
+with mathematica, so
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:6224]: Good catch!

**Comment by nelstrom** [comment]
Great work @jbcden! I've left a few comments on the PR but don't worry about responding to them. Consider them as notes-to-myself for changes I'll make after merging this PR.

**Comment by jbcden** [comment]
@nelstrom Would it be possible to include my changes to the John Wiegley in this PR too? If not I'll just wait until these changes are merged and try to do a merge.

**Comment by nelstrom** [comment]
@jbcden yeah, it looks as though GitHub wants to combine those two pieces of work into a single PR. That's not ideal, but it's no big problem. I'm happy to accept these changes in their current form.

**Comment by jbcden** [comment]
@nelstrom sorry about that. I will be more conscious of this in the future!

**Comment by nelstrom** [comment]
Same thing happened on [this PR](https://github.com/fromPeerToPeer/captions/pull/16/files). It probably won't be an issue again, since we're unlikely to have two sets of captions being proof-read simultaneously.

File: 3/ollie-charles-captions.srt (hunk 4 above)
Comment (nelstrom) [inline 3/ollie-charles-captions.srt:2308]: In Vim, this lovely regex matches captions that are not seperated from their predecessor with a newline:

```
\S\n\zs\d\+\n
```

**Pull request merged**